edge-currency-bitcoin/package.json

```diff
@@ -52,10 +59,57 @@
     "url-parse": "^1.4.3"
   },
   "devDependencies": {
+    "@babel/core": "^7.1.2",
```

Comment:
Now that this is in Lerna, all these tools should be at the top level.

Reply:
Still needs to be resolved